### PR TITLE
Separates mothpeople antennae into a mutant bodypart

### PR DESCRIPTION
**Pull request by ArcaneDefence**
After a bit of going crazy I decided to shoot myself and asked Potato for permission. He said it should be fine.

## About The Pull Request

Removes the antennae part of the original moth wing sprites and adds them back as a mutant bodypart. 
Seeing as they were then separated I made it a preference so that players can choose which antennae sprite they would like to have, instead of being restricted solely to the ones built into the wing sprite.

## Why It's Good For The Game

image
Mothpeople have always had atrocious dismemberment, because their antennae and markings are present only on the torso.
Here you can see the antennae and facial markings floating over where their head would be, post decapitation.
This is before my changes.

image image

This is after, with the original carbon present for comparison. 
The antennae are affected by fire the same way that their wings are, and have no mechanical function outside of that. 
Their antennae are visible again if their head is reattached, and are restored when ahealed just like burnt off wings are. 

## Changelog
:cl:
add: Mothpeople now have a preference to choose their antennae type from.
fix: Mothpeople will no longer have floating antennae when decapitated.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
wing organ that detaches upon decapitation when

**Comment by LemonInTheDark** [comment]
*Insert mild statement of approval here*

**Comment by ATH1909** [comment]
❤️ ❤️ ❤️ ❤️ ❤️ 

~~Separate neck fluff body part when????~~

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -1903,6 +1903,74 @@
...
+
+/datum/sprite_accessory/moth_antennae/whitefly
+	name = "White Fly"
+	icon_state = "whitefly"
+
+/datum/sprite_accessory/moth_antennae/lovers
+	name = "Lovers"
+	icon_state = "lovers"
+
+/datum/sprite_accessory/moth_antennae/punished
```
Comment (Jared-Fogle) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:1934]: Should document that this is the one used for Burnt Off.

**Comment by Timberpoes** [comment]
I'll allow it.

**Comment by kassori** [comment]
> I'll allow it. If only so I can pluck moth antennae now.

moster truck

**Review by Rohesie — APPROVED** [review]

**Comment by CoffeeDragon16** [comment]
What about fixing the antennae? There should be some way, as there is for Wing Reconstructive Surgery.

**Comment by ArcaneDefence** [comment]
> 
> What about fixing the antennae?

As it is you don't. I'm of the opinion that the wing burning mechanic has been made utterly trivial between body temperature changes and the reconstructive surgery. 
What once was a highly round defining semi-common occurrence has become rarer than appendicitis that takes just a little more effort to treat.

I'd rather it be even easier to burn, but that's starting to run a tangent. I don't care to add support for restoring antennae beyond what is already there for ahealing. Someone else can do that if they want it.

**Pull request merged**